### PR TITLE
Translated into Spanish.

### PR DESCRIPTION
**Pull request by guzman-diaz**

### CODE REVIEW

File: content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  from demographics as dm
+  left join adverse_events as ae
+  using(id)
+  where term <> 'Headache'
+")
+```
+
+Por cierto, el paquete  {dbplyr} traduce tu código  {dplyr} a SQL. De este modo no necesitas en realidad aprender SQL para consultar una base de datos.
+
+En este artículo hemos visto los cuatro modos más habituales de fusionar tablas y cómo implementarlos en R utilizando {base}, {dpyr} y SQL vía {tidyquery}. Teniendo este conocimiento, deberías ser capaz de fusionar con confianza cualquier conjunto de datos que te encuentres en R. Si te atascas, no obstante, no dudes en preguntar en los comentarios aquí abajo.
```
Comment (thomas-neitmann) [inline content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown:185]: Please add a sentence saying "This article has been kindly translated by <your name or twitter handle with link to website/twitter/GitHub (whatever you like)>".

File: content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,185 @@
+---
+title: Todo lo que necesitas saber sobre fusionar datos en R
+author: Thomas Neitmann
+date: '2021-02-01'
+slug: r-merging-datasets
```
Comment (thomas-neitmann) [inline content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown:5]: Could you translate the `slug` as well? This will become the URL of the post.

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]
Thank you so much! Please have a look at my minor comments 🙂

File: content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown (hunk 1 above)
Comment (guzman-diaz) [reply content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown:185]: I've just added the sentence at the end.

File: content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown (hunk 2 above)
Comment (guzman-diaz) [reply content/posts/2021-01-28-r-merging-datasets.es.Rmarkdown]: Done.

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]
Thank you so much!

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**